Library/Homebrew/requirements/blas_requirement.rb

```diff
@@ -0,0 +1,124 @@
+require "requirement"
+
+class BlasRequirement < Requirement
+  fatal true
+  # on OSX -lblas and -llapack should work OOB.
+  # the only case when test should fail is when we
+  # need BLAS with single precision or complex with Fortran =>
+  # use veclibfort
+  default_formula "veclibfort" if OS.mac?
```

Comment:
You're going to hit some issues with this as it doesn't know what `veclibfort` is because the CIs work off the assumption not everyone is going to have `homebrew/science` tapped. Same situation with `openblas`.

Until something happens on that the CI isn't going to be able to give you a huge amount of feedback I'm afraid.